components/resources/commands.grdp

```diff
@@ -241,6 +241,10 @@
   <message name="IDS_IDC_IMPORT_SETTINGS">
     Import settings
   </message>
+
+  <message name="IDS_IDC_OPEN_FULL_PAGE_CHAT">
+    Open AI Chat
```

Comment:
Should it be `Leo AI` ?

Reply:
yeah, thanks! Sorry I actually opened this PR only to get a CI build 😄